TODO.md

```
@@ -104,7 +104,7 @@
     - [x] Inject "on conflict do update"
     - [x] `check` function
     - [ ] Custom functions
-    - [ ] Accessing tables not in the schema
+    - [x] Accessing tables not in the schema
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)